packages/fxa-settings/src/components/ButtonDownloadRecoveryKey/index.tsx

```diff
@@ -86,31 +88,30 @@ export const ButtonDownloadRecoveryKey = ({
     const maxLength = 70;
     const prefix = 'Firefox-Recovery-Key';
     let email = primaryEmail.email;
-    let filename = `${prefix}_${date}_${email}`;
+    let filename = `${prefix}_${date}_${email}.txt`;
```

Comment:
Might be nice to have a little test coverage for both these cases.

Reply:
Thanks! Added some more test coverage for this.